Makefile

```diff
@@ -3,7 +3,7 @@
 # To re-generate a bundle for another specific version without changing the standard setup, you can:
 # - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
-DEFAULT_VERSION := 99.0.0
+DEFAULT_VERSION := 1.5.0
 VERSION ?= $(DEFAULT_VERSION)
 
```

Comment:
Also update CHANNELS and DEFAULT_CHANNEL to stable-1.5 ?

Reply:
I think we want stable here to match PROW and prod

Reply:
@shubham-pampattiwar Should just be stable, since we're using stable for this release rather than stable-1.5 -- we had a discussion yesterday on slack about that. @weshayutin Right?

Reply:
`stable`

Reply:
Yes. `stable` here reflects shipping channel. Although this really only affects `make deploy-olm` operator.